.mailmap

```diff
@@ -542,3 +542,4 @@ Zachary Vacura <admin@hackzzila.com>
 Zoran Tomicic <ztomicic@gmail.com>
 Сковорода Никита Андреевич <chalkerx@gmail.com>
 隋鑫磊 <joshuasui@163.com>
+Jacob Smith <3012099+JakobJingleheimer@users.noreply.github.com>
```

Comment:
The mailmap entries are in alphabetical/lexical/codepoint order (which I find helpful to avoid duplicate/conflicting entries). I'll add a fixup commit to move this entry.

Reply:
🙇‍♂️